C/c4BlobStore.cc

```diff
@@ -282,29 +279,25 @@ void C4BlobStore::replaceWith(C4BlobStore &other) {
 
 
 C4ReadStream::C4ReadStream(const C4BlobStore &store, C4BlobKey key)
-:_impl(store.getReadStream(key))
-{ }
+        : _impl(store.getReadStream(key)) {}
 
-C4ReadStream::C4ReadStream( C4ReadStream &&other)
-:_impl(move(other._impl))
-{ }
+C4ReadStream::C4ReadStream(C4ReadStream &&other)
+        : _impl(move(other._impl)) {}
 
 C4ReadStream::~C4ReadStream() = default;
-size_t C4ReadStream::read(void *dst, size_t mx)     {return _impl->read(dst, mx);}
-int64_t C4ReadStream::getLength() const             {return _impl->getLength();}
-void C4ReadStream::seek(int64_t pos)                {_impl->seek(pos);}
+
```

Comment:
In these single-line inline methods, if the method bodies aren't lined up they get a lot harder to read. I was really meticulous about making these line up nicely.

Also, adding blank lines between them doesn't seem to add anything (they're one per line so it's already clear where the separations are), and it takes up more vertical space so you can't see as much onscreen.

Reply:
Unfortunately, I couldn't find an option to do this automagically with Clang-Format. Luckily, you can just disable Clang-Format for sections of code, so I've returned your code to it's previous beautiful alignment, and added `// clang-format off` for that section of code.